test/regression/sql/cte.sql

```diff
@@ -6,3 +6,27 @@ WITH modifying_cte AS (
 ) select * from modifying_cte;
 
 DROP TABLE t;
+
+WITH RECURSIVE outermost(x) AS (
```

Comment:
nit: can you move this query before the `DROP TABLE` command. I think it's nice for consistency between tests to keep all the table drops at the end of the sql file.

Reply:
Ok, that's done